dbus-mqtt-grid/dbus-mqtt-grid.py

```diff
@@ -173,12 +173,12 @@ def on_message(client, userdata, msg):
                             grid_forward = (
                                 float(jsonpayload["grid"]["energy_forward"])
                                 if "energy_forward" in jsonpayload["grid"]
-                                else 0
+                                else None
```

Comment:
This is needed to not reset the energy_forward to 0, if a "power" reading is received that does not contain energy_forward